src/components/Sections/SectionHeader.tsx

```diff
@@ -36,9 +36,9 @@ export const SectionHeader = ({
             {section?.name}
           </Divider>
         ) : isCategory ? (
-          t(`category.${section.name}`)
+          <div className="u-ellipsis">{t(`category.${section.name}`)}</div>
```

Comment:
Is there any reason why there shouldn't be a margin on the category title?

Reply:
because there is no button yes